apps/autonolas-registry/components/Layout/styles.jsx

```diff
@@ -29,6 +30,20 @@ export const CustomLayout = styled(Layout)`
     }
   }
 
+  /* table */
+  .ant-table {
+    .ant-table-tbody > tr {
+      > td {
+        padding: 8px 16px;
+        .ant-btn {
+          &:first-child {
+            padding-left: 0;
```

Comment:
struggled with in Govern and wanted to suggest moving to shared libs but realise we have it here. do you have any idea why it doesn't work from there?